app/src/main/java/org/listenbrainz/android/util/Constants.kt

```diff
@@ -12,6 +12,7 @@ object Constants {
     const val FEEDBACK_SUBJECT = "[LBAndroid] Feedback"
     const val SPOTIFY_REDIRECT_URI = "org.listenbrainz.android://callback"
     const val LISTENBRAINZ_API_BASE_URL = "https://api.listenbrainz.org/1/"
+    const val LISTENBRAINZ_BETA_API_BASE_URL = "https://beta-api.listenbrainz.org/1/"
```

Comment:
Mark as to be removed when [this](https://github.com/metabrainz/listenbrainz-android/pull/317) pr is merged.